### ISSUE TITLE
[Feature Request] Don't separate Album by year 

### ISSUE THREAD

**Issue by UiharuKazari2008** [ampache/ampache#2247]
It would be nice if the album list would not break up albums that have tracks from multiple years and handles them the same way that iTunes would handle it as a single album and would only separate a album by name / album artist.

<img width="1469" alt="Screen Shot 2019-12-20 at 14 49 00" src="https://user-images.githubusercontent.com/15165770/71288018-e62b6b00-2337-11ea-86e7-7efe244f2aa6.png">

**Comment by lachlan-00** [comment]
The year is meant to be album year even for compilations.

The individual songs could use tags for original year or something else but even original year is meant to show the original release year for rereleased albums

**Comment by skriker** [comment]
Check the option "Album - Group multiple disks" in your user settings - I believe it could fix your problem. Please note that you can change this setting on per user basis.

**Comment by WisdomCode** [comment]
@striker that unfortunately doesn't seem to fix it (at least for me)

I don't know how its meant to be, but it can be complicated to make the years teared apart into a tag and a value of the album.
This would mean that you could search for a specific year, or make a smart playlist containing the songs of a certain decade and missing out then on the songs from compilations and Best Ofs (and including Compilations of songs that have nothing to do with that year if this album was released later). This doesnt seem like something intended to me, and as iTunes, banshee and rhythmbox aren't following that route (making importing your songs more difficult), maybe this should be made optional.

**Comment by lachlan-00** [comment]
Pretty much all online resources manage year as an album tag and not a song tag. All date tags are related to the street date of the release itself and compilations are done the same way.

Albums do have a tag for original year in musicbrainz which i've added into the database. my intention there is to have that used in search for year when available. (ie albums re-released in 2020 but originally released in 1980 can be searchable by that instead.)

I know itunes has the compilation tag but it's not standard. it might be a good idea it be able to distinguish those albums though with something like that. In the mean time it can be imported if you allow custom metadata right now. (verify will import all the extra tags not in use into the metadata table)

so TLDR, year, original_year, etc are not song tags but it doesn't stop custom tags being imported and searching them (like the itunes' compilation) using the custom_metadata option.

**Comment by lachlan-00** [comment]
Marking this as an enhancement/wontfix. I think it's valid in some ways but even in your example the release date is 2009 meaning it's should be tagged for 2009.
https://www.discogs.com/Various-Best-Of-German-Hands-Up/release/1834526

I won't change the current default right now to allow this but that's not to say it can't be done with a bit of work. we could put in compilation tags using the itunes method or something similar in a different tag.
